spec/marc_to_solr/lib/princeton_marc_spec.rb

```diff
@@ -155,7 +155,7 @@ def fixture_record(fixture_name)
       let(:l856) { { "856" => { "ind1" => " ", "ind2" => " ", "subfields" => [{ "u" => url }, { "x" => "test text3" }] } } }
 
       it 'retrieves the URLs and the link labels' do
```

Comment:
issue: The wording of this should change.

Reply:
oh right @eliotjordan . will update the title.